### DISCUSSION TITLE
Write Medium Articles for Stellar Quest Challenges

### DISCUSSION THREAD

**Discussion by apaldiwal** [kalepail/stellar-quest-bounties#95] [Bounty Ideas]
Is it a good idea to make a bounty on writing walkthrough articles on medium for Stellar Quest challenges? We can make a `stellar-quest-walkthrough` repository where we can create and host articles and whatever income that is being generated by writing the articles can be utilized for future bounties. I know it really won't be a massive amount of income, but I want the bounty program to be self-sustainable.

**Comment by ElliotFriend** [comment]
I think that could be a pretty valuable thing. Having high-quality explanations of how some of these operations are performed could be incredibly beneficial to many of those who ask in the discord "Can I do this if I'm not a programmer." That kind of information (especially for some of the earlier quests) could really help ease folks into the swing of things. An "on-ramp" kinda thing.

Might be useful to have video walkthroughs as a bounty, as well?

**Reply by apaldiwal** [reply]
Yeah, it's a pretty solid idea! Maybe we can combine articles + videos in the bounty?

**Comment by kalepail** [comment]
I love this idea! We could have a whole series and then maybe even link to them from within the Stellar Quest practice quests themselves. Could have bounties for writing and for doing video walk throughs. 

In general I love the idea of supporting non-engineering only bounties. Also there's no need to host these all in one place. People could write them on their own accounts on medium or really anywhere. Spreading SQ content far and wide would be good and then we can just link to it here in the bounties repo and for the really good ones we can link directly from within the SQ app.
